diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache GNUDoku-0.93-orig/CYGWIN-PATCHES/README GNUDoku-0.93/CYGWIN-PATCHES/README
--- GNUDoku-0.93-orig/CYGWIN-PATCHES/README	1969-12-31 18:00:00.000000000 -0600
+++ GNUDoku-0.93/CYGWIN-PATCHES/README	2006-01-05 14:32:11.640625000 -0600
@@ -0,0 +1,55 @@
+gnudoku
+------------------------------------------
+GNUDoku is a free program for creating and solving Su Doku puzzles.
+
+Runtime requirements:
+  cygwin-1.5.18-1
+  gtkmm24-2.6.5-1
+
+Build requirements:
+(besides corresponding -devel packages)
+  cygwin-1.5.18-1
+  binutils-20050610-1
+  gcc-core-3.4.4-1
+  make-3.80-1
+  pkgconfig-0.17.2-2
+
+Canonical website:
+  http://icculus.org/~jcspray/GNUDoku/
+
+-------------------------------------------
+
+Build instructions:
+  unpack gnudoku-0.93-X-src.tar.bz2
+    if you use setup to install this src package, it will be
+	 unpacked under /usr/src automatically
+  cd /usr/src
+  ./GNUDoku-0.93-X.sh all
+
+This will create:
+  /usr/src/gnudoku-0.93-X-src.tar.bz2
+  /usr/src/gnudoku-0.93-X.tar.bz2
+
+-------------------------------------------
+
+Files included in the binary package:
+
+  /usr/bin/gnudoku.exe
+  /usr/share/applications/GNUDoku.desktop
+  /usr/share/doc/Cygwin/gnudoku-0.93.README
+  /usr/share/doc/gnudoku-0.93/ChangeLog
+  /usr/share/doc/gnudoku-0.93/COPYING
+  /usr/share/doc/gnudoku-0.93/README
+  /usr/share/doc/gnudoku-0.93/TODO
+  /usr/share/pixmaps/GNUDoku.png
+
+------------------
+
+Port Notes:
+
+----- version 0.93-1 -----
+Initial release for Cygwin.
+
+
+Cygwin port maintained by:
+http://cygwinports.dotsrc.org/
diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache GNUDoku-0.93-orig/CYGWIN-PATCHES/setup.hint GNUDoku-0.93/CYGWIN-PATCHES/setup.hint
--- GNUDoku-0.93-orig/CYGWIN-PATCHES/setup.hint	1969-12-31 18:00:00.000000000 -0600
+++ GNUDoku-0.93/CYGWIN-PATCHES/setup.hint	2006-01-05 14:32:11.703125000 -0600
@@ -0,0 +1,4 @@
+category: Gnome Games
+requires: cygwin gtkmm24 xorg-x11-base
+sdesc: "Su Doku puzzle creator and solver"
+ldesc: "GNUDoku is a free program for creating and solving Su Doku puzzles."
diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache GNUDoku-0.93-orig/GNUDoku.desktop GNUDoku-0.93/GNUDoku.desktop
--- GNUDoku-0.93-orig/GNUDoku.desktop	2005-12-11 12:21:29.000000000 -0600
+++ GNUDoku-0.93/GNUDoku.desktop	2006-01-05 14:19:20.453125000 -0600
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=GNUDoku
 Comment=A program for creating and solving Su Doku puzzles
-Exec=GNUDoku
+Exec=gnudoku
 Icon=GNUDoku.png
 Terminal=false
 Type=Application
diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache GNUDoku-0.93-orig/Makefile GNUDoku-0.93/Makefile
--- GNUDoku-0.93-orig/Makefile	2005-12-11 12:21:30.000000000 -0600
+++ GNUDoku-0.93/Makefile	2006-01-05 14:19:07.703125000 -0600
@@ -1,15 +1,15 @@
 
-PREFIX=/usr/local
+PREFIX=/usr
 BINDIR=$(PREFIX)/bin
 
 VERSION=\"0.93\"
 
 SOURCES=GNUDoku.C sudoku.C sudoku-solve.C
-TARGET=GNUDoku
+TARGET=gnudoku.exe
 CXXFLAGS=`pkg-config gtkmm-2.4 --cflags` "-DVERSION=$(VERSION)"
 LIBS=`pkg-config gtkmm-2.4 --libs`
 
-TEST_TARGET=GNUDoku-test
+TEST_TARGET=gnudoku-test.exe
 TEST_SOURCES=sudoku-solve.C sudoku-solve-test.C
 TEST_CXXFLAGS=-DSUDOKU_SOLVE_DEBUG=true -g
 
@@ -22,12 +22,12 @@
 	$(CXX) $(SOURCES) -o $(TARGET) $(CXXFLAGS) $(LIBS)
 
 install:
-	install -d $(BINDIR)
-	install -s $(TARGET) $(BINDIR)
-	install -d $(PREFIX)/share/applications
-	install -m 0644 GNUDoku.desktop $(PREFIX)/share/applications
-	install -d $(PREFIX)/share/pixmaps
-	install -m 0644 GNUDoku.png $(PREFIX)/share/pixmaps
+	install -d $(DESTDIR)$(BINDIR)
+	install -s $(TARGET) $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(PREFIX)/share/applications
+	install -m 0644 GNUDoku.desktop $(DESTDIR)$(PREFIX)/share/applications
+	install -d $(DESTDIR)$(PREFIX)/share/pixmaps
+	install -m 0644 GNUDoku.png $(DESTDIR)$(PREFIX)/share/pixmaps
 
 uninstall:
 	rm -f $(BINDIR)/$(TARGET)
